common/djangoapps/custom_settings/tests/test_models.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+from .factories import CustomSettingsFactory
+
+
+@pytest.mark.django_db
+def test_save_custom_settings_and_no_prior_course_short_id():
+    """Assert that course short id is set to 100, when CustomSettings is saved"""
+    custom_settings = CustomSettingsFactory(course_short_id=None)
+    assert custom_settings.course_short_id == 100
```

Comment:
Just curious, How it would be 100? aren't we getting a random int for this?

Reply:
We are taking random int for unit test, but in actual code, `course_short_id` can be equal to 100.

See `models.py` at `edx-platform/common/djangoapps/custom_settings/models.py:40`

Reply:
Shouldn't we write another test case to check if we add two custom settings, It should be 101 for second custom settings?

Reply:
You already tested that case 👍